docs/configs/pdf/index.rst

```diff
@@ -99,6 +106,8 @@ Early Access Features
    daml-integration-kit/index
    triggers/index
    tools/visual
+   tools/trigger-service
```

Comment:
We really need to unify the two indices.

Reply:
I would if I knew how...

Reply:
I’ll take a look